src/components/HdTagsList.vue

```diff
@@ -23,18 +32,36 @@ export default {
 @import 'homeday-blocks/src/styles/mixins.scss';
 
 .tags-list {
+  $root: &;
   display: flex;
   flex-wrap: wrap;
   min-height: #{$sp-l + $sp-s};
   margin-bottom: $sp-m;
+
   &__tag {
     display: flex;
     align-items: center;
-    height: $sp-l;
-    background-color: getShade($quaternary-color, 50);
-    border-radius: 3px;
     padding: 0 $sp-s;
     margin: $sp-s $sp-s 0 0;
   }
+
+  &--primary {
+    #{$root}__tag {
+      height: $sp-l;
+      background-color: getShade($quaternary-color, 50);
+      border-radius: 3px;
+    }
+  }
+  &--secondary {
+    #{$root}__tag {
+      height: $sp-m;
+      background-color: $cello-blue;
```

Comment:
[abstraction](https://github.com/homeday-de/homeday-blocks/blob/master/src/styles/_colors.scss#L26)
```suggestion
      background-color: $primary-color;
```